### PR TITLE
Fixes and improvements

### LINKED ISSUE #39: Rename everything to Test -> Experiment -> Trials
This issue is a continuation of [issue #48 from drivers-maker](https://github.com/benchflow/drivers-maker/issues/48). 

We should remove from `experiments-manager` all references to `benchflowBenchmark` or `benchmarkName`, and replace them by `benchflowTest/benchflowExperiment` and `experimentName`.

### PR DESCRIPTION
**Pull request by simonedavico**
- Applies consistent naming for test/experiment/trial across project source (closes #39);
- Removes `/deploy` API;
- Removes `BenchFlowMinioClient` sources;
- Updates CI scripts.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/benchflow/experiments-manager/pull/40%23discussion_r72663965%22%2C%20%22https%3A//github.com/benchflow/experiments-manager/pull/40%23discussion_r72665386%22%2C%20%22https%3A//github.com/benchflow/experiments-manager/pull/40%23discussion_r72671758%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20e1ba95c387f60def7a53561526ae36a0774871c6%20application/pom.xml%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/experiments-manager/pull/40%23discussion_r72663965%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Open%20an%20issue%20similar%20to%20https%3A//github.com/benchflow/drivers-maker/issues/54%22%2C%20%22created_at%22%3A%20%222016-07-28T17%3A21%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22Addressed%20in%20issue%20%2341%20%22%2C%20%22created_at%22%3A%20%222016-07-28T17%3A30%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4446853%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/simonedavico%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-07-28T18%3A06%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20application/pom.xml%3AL89-101%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull e1ba95c387f60def7a53561526ae36a0774871c6 application/pom.xml 4'></a> &nbsp;<b><a href='https://github.com/benchflow/experiments-manager/pull/40#discussion_r72663965'>File: application/pom.xml:L89-101</a></b>
- <a href='https://github.com/VincenzoFerme'></a> Open an issue similar to https://github.com/benchflow/drivers-maker/issues/54
- <a href='https://github.com/simonedavico'></a> Addressed in issue #41

<a href='https://www.codereviewhub.com/benchflow/experiments-manager/pull/40?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/benchflow/experiments-manager/pull/40?approve=1'></a>&nbsp;<a href='https://github.com/benchflow/experiments-manager/pull/40'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by VincenzoFerme** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/benchflow/experiments-manager/pull/40'>Refresh</a>.<a href='#crh-update-none'></a>

File: application/pom.xml [hunk 1]
```diff
@@ -89,6 +89,12 @@
             <version>${gson.version}</version>
         </dependency>
 
+		<dependency>
```
Comment (VincenzoFerme) [inline application/pom.xml]: Open an issue similar to https://github.com/benchflow/drivers-maker/issues/54

Comment (simonedavico) [inline application/pom.xml]: Addressed in issue #41

Comment (VincenzoFerme) [inline application/pom.xml]: :+1:

**Pull request merged**